src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -98,7 +98,7 @@ class AuthScreens extends React.Component {
         Pusher.init({
             appKey: CONFIG.PUSHER.APP_KEY,
             cluster: CONFIG.PUSHER.CLUSTER,
-            authEndpoint: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=AuthenticatePusher`,
+            authEndpoint: `${CONFIG.EXPENSIFY.DEFAULT_API_ROOT}api?command=AuthenticatePusher`,
```

Comment:
A "hidden" thread about this: https://github.com/Expensify/App/pull/15178#discussion_r1126939927

TL;DR 

staging is using DEFAULT_API_ROOT, which on staging is the prod API - www.expensify.com and not the staging api - staging.expensify.com

Reply:
I think we can just leave this so gonna resolve this one.